runtime/src/main/java/com/tietoevry/quarkus/resteasy/problem/validation/ConstraintViolationExceptionMapper.java

```diff
@@ -31,14 +31,20 @@
 @Priority(Priorities.USER)
```

Comment:
This priority annotation is very **miss-leading** in case of  ``resteasy-reactive``, as the extension overrides the prioarty at build time: https://github.com/TietoEVRY/quarkus-resteasy-problem/blob/d253adac319b8422944159d3bfdeb7d1db91eda5/deployment/src/main/java/com/tietoevry/quarkus/resteasy/problem/deployment/ProblemProcessor.java#L115C1-L115C1

Reply:
All I can say is - one does not simply create extension that is compatible with jackson/jsob + classic/reactive matrix ;)

All I can do here is to add clarifying comment, because the tradeoff of `the clean way` would be to have separate mappers for classic and reactive, which would look almost exactly the same.

Reply:
Look at [this comparison](https://github.com/TietoEVRY/quarkus-resteasy-problem/wiki) to see what we're trying to unify with this extension ;)